modules/javafx.graphics/src/main/native-glass/mac/GlassWindow.m

```diff
@@ -843,7 +843,9 @@ static jlong _createWindowCommonDo(JNIEnv *env, jobject jWindow, jlong jOwnerPtr
         }
         else
         {
-            [window->nsWindow performSelectorOnMainThread:@selector(setContentView:) withObject:nil waitUntilDone:YES];
+            // If the contentView is set to nil within performKeyEquivalent: the OS will crash.
+            NSView* dummy = [[NSView alloc] initWithFrame: NSMakeRect(0, 0, 10, 10)];
```

Comment:
The view could be also created with a (0, 0, 0, 0) rect, couldn't it?

Reply:
I don't know. I'm always skeptical of 0 size rectangles. I'd either leave it as-is or maybe use a `(0,0,1,1)` rectangle.

Reply:
The initial size should not matter since it will be re-sized to the window's content bounds. But I would rather avoid doing something unusual like creating a 0-sized NSView even though it almost certainly will work.

Reply:
Okay, no problem then.